src/Sql/Sql/AdvancedThreatProtection/Cmdlet/DisableAzureSqlManagedInstanceAdvancedDataSecurity.cs

```diff
@@ -31,7 +32,7 @@ public class DisableAzureSqlManagedInstanceAdvancedDataSecurity : SqlManagedInst
         protected override ManagedInstanceAdvancedDataSecurityPolicyModel PersistChanges(ManagedInstanceAdvancedDataSecurityPolicyModel model)
         {
             model.IsEnabled = false;
-            ModelAdapter.SetManagedInstanceAdvancedDataSecurity(model);
+            ModelAdapter.SetManagedInstanceAdvancedDataSecurity(model, DefaultContext.Environment.GetEndpoint(AzureEnvironment.Endpoint.StorageEndpointSuffix));
```

Comment:
Do we have test coverage for this?  Also, please update the changelog in src\Sql\Sql  for this